src/color-system.mdx

```diff
@@ -47,6 +47,35 @@ We have not used HEX values to create the CSS variables for each color. They are
 Thus, to get a solid blue color, you would write `color: rgb(var(--color-blue-default));`.
 To get a blue color with 10% opacity, write `color: rgba(var(--color-blue-default), 0.1);` instead.
 
+### Customizing our components and your apps with colors
+When using web-components from Lime Elements, you will notice that there are many components which allow you to add colors. However, since Lime Elements is a design system intended to unify the look and feel of our products, we naturally try to reduce your styling choices. So this color customization will enable you to either add an accent color (e.g. your product color) to the component, or add colors that contribute the the functionality of the component (e.g. indicating a status such as error, danger, etc).
+
+But when you do your own add-ons, integrations, and apps, keep in mind that the UI should be neutrally colored (using the contrast scales). Applying intensive colors to big areas of the UI (such as headers and background colors) is not recommended.
+
+Colors should be used to communicate meaning, and attract attention to things that are functionally important; not to scream your brand throughout the entire product! Different sections of the app should be distinct, but also blend together well. Extensive color usage will be placing the emphasis on an app’s UI structure, instead of its content.
+
+Use only one accent color (product color) and make sure that it consistently affects all components you use from lime elements in your product. This value is called `--lime-primary-color`. We don't recommend using the brand color css variables directly in this case, but instead suggest using their equivalent swatch form the color palette. This will ensure that your UI will automatically benefit from dark mode adjustments, while users do not notice the difference.
+
+Things that can be affected by the accent color:
+* In buttons: background color of primary buttons
+* In check boxes: background color of “checked” boxed
+* In radio buttons: background color of “checked” buttons
+* In inputs: the underline visual effect
+* In linear progress: color of the progressed section and what’s left (its background)
+* In many components: the visual styles that indicate the `focused` state (using keyboard)
+* In many components: the visual styles that indicate the `active` or `selected` state
+* etc...
+
+What colors can you optionally change to communicate meaning
+* In buttons: text color, and also background color of buttons with certain functionalities (delete, restore, etc..)
+* In badges: color of the page and its test color
```

Comment:
Is this supposed to say "and it's text color"? Otherwise I don't understand what this means 😄

Reply:
:D WTF, yes... its text color

Reply:
:zap: